content/tracing/faq/traces-sampling-and-storage.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Traces sampling and storage
+kind: documentation
+---
+
+## Trace sampling 
+
+Due to the extremely high volume of traces in a web-scale application, sampling is applied to traces in Datadog.  
+**Statistics (requests, errors, latency, etc.) are calculated based on the full volume of traces at the agent level, and are therefore always accurate**.
+
+### Statistics (Requests, Errors, Latencies etc.)
+
+Datadog APM computes aggregate statistics over all the traces instrumented, regardless of sampling:
+
+* Total requests and requests per second
+* Total errors and errors per second
+* Latency
+* Breakdown of time spent by service/type
+* [Apdex score](/tracing/faq/how-to-configure-an-apdex-for-your-traces-with-datadog-apm) (web services only)
+
+{{< img src="tracing/faq/sampling_stats.png" alt="Aggregate statistics are generated on un-sampled data." responsive="true" popup="true" style="width:90%;">}}
+
+### Traces
+
+To ensure keeping a representative sample set of traces, Datadog combines multiple sampling techniques, at various locations: 
```

Comment:
+To ensure a representative sample set of traces are kept, Datadog combines multiple sampling techniques, at various locations:

Reply:
👍